truthy-falsy.js

```diff
@@ -0,0 +1,117 @@
+// 1
+const isEmployed = true;
+const isOnVacation = false;
+
+function setAlarm(isEmployed, isOnVacation){
```

Comment:
You shouldn't use the `isEmployed` and `isOnVacation` names here since you already have variables with the same name available here. Either change the argument names, change the variable names, or get rid of the variables.